src/WebJobs.Script/Content/Script/functions.js

```diff
@@ -69,6 +70,11 @@ function createFunction(f) {
         inputs.unshift(context);
         delete context._inputs;
 
+        if (context._http) {
```

Comment:
sounds reasonable.  I was originally thinking more in terms of output bindings, but as there's 1:1 mapping for httptrigger -> httpoutputbinding this work